int32 b
---
int32 power_percent_new
